sound_main.py

```diff
@@ -0,0 +1,26 @@
+
+from selenium import webdriver
+import time
+
+driver = webdriver.Chrome('D:\Новая папка\chromedriver.exe')

```

Comment:
@Volley97 it is much better to specify instructions inm a readme on how to configure chromedriver(add it to PATH var) instead of hardcoding the path.